soc/riscv/riscv-privilege/starfive_jh71xx/Kconfig.soc

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "RISC-V Starfive JH7100 SoC"
```

Comment:
RISC-V is obvious so "Starfive JH7100 SoC" is better.

Reply:
Fixed.
Thanks.

Reply:
Resolving as @katsuster has already OK'ed